tests/test_app.py

```diff
@@ -93,6 +93,107 @@ def init_app(request):
     else:
         return firebase_admin.initialize_app(CREDENTIAL)
 
+@pytest.fixture(scope="function")
+def env_test_case(request):
+    config_old = set_config_env(request.param.config_json)
+    def fin():
```

Comment:
Can we use `yield ` here? http://pytest.readthedocs.io/en/2.8.7/yieldfixture.html